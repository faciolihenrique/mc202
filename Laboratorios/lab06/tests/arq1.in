# Expressões simples sem parênteses
a
-a
+a
a+b
a-b
a*b
a/b
a^b
