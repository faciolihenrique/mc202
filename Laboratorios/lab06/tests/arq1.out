# Expressões simples sem parênteses
Infixa: a
Prefixa: a
Posfixa: a


Infixa: -a
Prefixa: ~a
Posfixa: a~


Infixa: +a
Prefixa: &a
Posfixa: a&


Infixa: a+b
Prefixa: +ab
Posfixa: ab+


Infixa: a-b
Prefixa: -ab
Posfixa: ab-


Infixa: a*b
Prefixa: *ab
Posfixa: ab*


Infixa: a/b
Prefixa: /ab
Posfixa: ab/


Infixa: a^b
Prefixa: ^ab
Posfixa: ab^


Processamento terminado
